src/components/NavBar/AccountMenu.tsx

```diff
@@ -1,18 +1,35 @@
 import { useAuth0 } from "@auth0/auth0-react";
-import { Avatar, Link } from "@mui/material";
+import { Avatar, Box, Link } from "@mui/material";
 import { Typography } from "@mui/material";
 import { Button } from "@mui/material";
 import Popover from "@mui/material/Popover";
 import Stack from "@mui/material/Stack/Stack";
 import React from "react";
 import { useAppSelector } from "../../state/hooks";
+import LogoutIcon from "@mui/icons-material/Logout";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
 
 const styles = {
   button: {
     marginTop: "auto",
     marginBottom: 2,
   },
   nameContainer: { paddingLeft: 2, textAlign: "left" },
+  popover: {
+    paddingTop: "10px",
+    paddingBottom: "10px",
+    borderRadius: "10px",
+  },
+  popoverButton: {
+    paddingTop: "5px",
+    paddingBottom: "5px",
+    paddingRight: "20px",
+    paddingLeft: "20px",
+    textTransform: "none",
+  },
+  popoverIcon: {
+    marginRight: "10px",
```

Comment:
Same here, can we make this 
```suggestion
    marginRight: 2,
```

Reply:
Done